src/index.html

```diff
@@ -136,27 +150,25 @@ <h2>Question</h2>
           />
         </button>
         <h2>Question</h2>
-        <details>
-          <summary>Show answer</summary>
-          Insert answer here<br />
+          <button data-js="button" class="answer__button">Show answer</button>
+          <p data-js="answer" class="hidden">Insert answer here</p><br />
           <button>Tag 1</button>
           <button>Tag 2</button>
           <button>Tag 3</button>
           <button>Tag 4</button>
-        </details>
       </section>
     </section>
 
     <section data-js="page3" class="hidden app-grid__main">
       <form class="create">
         <label for="question">Question:</label><br />
-        <input type="text" id="question" /><br />
+        <textarea type="text" maxlength="200" data-js="textarea" nKeypress="zaehlen()"id="question" rows="4" cols="50"></textarea><br />
```

Comment:
Wenn du die textarea in das label verschachtelst kannst du dir id= und for= sparen.